build 2.8 (android): default avatar throws error in avatar library manager

It seems the default avatar is missing something, as it throws an error: 
```
ExceptionManager: [Error] [ERROR] [MirageXR.AvatarLibraryManager] Avatar with ID 667bed8204fd145bd9e09f19 is not in the library., trace: UnityEngine.DebugLogHandler:Internal_Log(LogType, LogOption, String, Object)
```
see https://wekit-ecs.sentry.io/issues/6949065987/events/8724e85c21564f67bc2e8aed2cf5411e/?project=4506320009428992

--- a/Assets/MirageXR/ReadyPlayerMe/Scripts/Loading/AvatarLibraryManager.cs
+++ b/Assets/MirageXR/ReadyPlayerMe/Scripts/Loading/AvatarLibraryManager.cs
@@ -33,7 +33,8 @@ namespace MirageXR
 			}
 			else
 			{
-				AvatarList = new List<string>() { AvatarLoader.DefaultAvatarUrl };
+				AvatarList = new List<string>();
+				AddAvatar(AvatarLoader.DefaultAvatarUrl);
 			}
 			_cachedAvatarThumbnails.Clear();
 		}

